integreat_cms/cms/utils/filter_links.py

```diff
@@ -47,6 +47,15 @@ def filter_links(region_slug, link_filter=None):
         for translation in translations
         for link in translation.links.all()
     ]
+    # Define found link types that are not supposed to be shown in the List (e.g. phone numbers and Emails)
+    ignored_link_types = [
+        "Phone number (not automatically checked)",
+        "Email link (not automatically checked)",
+    ]
+    links = [
+        link for link in links if link["link"].url.message not in ignored_link_types
+    ]
```

Comment:
I wouldn't rely on the hardcoded messages here since they might change in the future. Instead, we can also query for the type.
Also, we can include this filter in the previous loop, so we don't need to loop over all link twice.
I cannot make an automated suggestion, because it's out of range, but this would be my preferred solution:
```
    # Define found link types that are not supposed to be shown in the List (e.g. phone numbers and Emails)
    ignored_link_types = [
        "mailto",
        "phone",
        "anchor",
    ]
    # Convert translations to list of dicts to enable filtering of links in python
    links = [
        {"translation": translation, "link": link}
        for translation in translations
        for link in translation.links.all()
        if link.url.type not in ignored_link_types
    ]
```